src/vllm_tgis_adapter/__main__.py

```diff
@@ -89,11 +133,15 @@ async def create_chat_completion(
     request: ChatCompletionRequest,
     raw_request: fastapi.Request,
 ) -> JSONResponse:
-    generator = await openai_serving_chat.create_chat_completion(request, raw_request)
+    generator = await openai_serving_chat.create_chat_completion(
+        request,
+        raw_request,
+    )
```

Comment:
It would be good to keep the formatting the same as the upstream `openai_server.py` that this is a copy of, so that we can diff them easily.

I guess we are hoping to get rid of the duplication soon anyhow? In which case I guess it doesn't matter so much